app/controllers/admin/dashboard_controller.rb

```diff
@@ -10,5 +10,6 @@ def show
   def update
     order = Order.find(params[:id])
     order.ship
+    redirect_to '/admin'
```

Comment:
Nice, immediately see that it's in the shipped section. Need to go in and add that to the test.